wordpress-mobile/WordPress-Android
# Feature/site media settings images and videos

This is the first PR which implements the media settings redesign as per #6606 . In this PR, the new settings screen & fullscreen view have been implemented for images and videos. Settings for audio & documents will be implemented in subsequent PRs.

Note that a design review will come after all the PRs have been merged into the feature branch. That said, feel free to comment on any obvious design issues.

To test:

* Open the site media browser
* Tap an image
* Notice the snazzy new screen
* Tap the FAB
* Notice the new fullscreen preview

![2017-09-14 17_18_22](https://user-images.githubusercontent.com/3903757/30456221-c312a230-9970-11e7-800e-729d984ae952.gif)

Higher quality video demo [here](https://cloudup.com/cTa-x_B4i8z).
